get-started-microservices-on-kubernetes/simple-microservice-rest-client/simple-microservice-client-secured/src/main/java/org/wildfly/examples/GettingStartedEndpointInterface.java

```diff
@@ -0,0 +1,21 @@
+package org.wildfly.examples;
+
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.HeaderParam;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import org.eclipse.microprofile.rest.client.annotation.RegisterClientHeaders;
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+@RegisterClientHeaders
+@RegisterRestClient(configKey="simple-microservice-server")
+@Path("/hello")
+public interface GettingStartedEndpointInterface {
```

Comment:
IMO we should just name this `GettingStartedEndpointClient`.

Reply:
@jamezp I agree with you but I'd prefer to keep this naming because, otherwise, the project name and the interface name would have a similar naming which might result confusing (project `simple-microservice-client` would contain interface `GettingStartedEndpointClient`) ... but if you prefer that naming I'll change it